content/about/faq.md

```diff
@@ -102,8 +102,8 @@ menu: "footer"
 
     Yes, there are two libraries we heavily rely on and which we point to quite often when people complain about darkable lacking something:
 
-    * **exiv2** is used for reading metadata from image files. If something isn't shown correctly in the [image information](/usermanual/en/image_information.html) panel on the left side then please check with the command line tool `exiv2` and report any problems upstream on [their bug tracker](https://github.com/Exiv2/exiv2/issues)&nbsp;– there isn't much we can do to fix those things ourselves.
-    * **lensfun** is used for lens correction. If the [lens correction](/usermanual/en/correction_group.html#lens_correction) module isn't showing your camera or lens, or a wrong one, then please report that to [those folks](https://github.com/lensfun/lensfun).
+    * **exiv2** is used for reading metadata from image files. If something isn't shown correctly in the [image information](https://docs.darktable.org/usermanual/4.0/en/module-reference/utility-modules/darkroom/image-info-line/) panel on the left side then please check with the command line tool `exiv2` and report any problems upstream on [their bug tracker](https://github.com/Exiv2/exiv2/issues)&nbsp;– there isn't much we can do to fix those things ourselves.
```

Comment:
This should point to `https://docs.darktable.org/usermanual/4.0/en/module-reference/utility-modules/shared/image-information/`

Reply:
Fixed the link target. Thank you!